### ISSUE TITLE
Problem with Contenteditable div

### ISSUE THREAD

**Issue by dineshi2it** [jasonslyvia/react-anything-sortable#38]
I'm in a situation where I need to render a contenteditable div.

So, just tried it by adding contenteditable div in demoItem.js. Then, whenever I click inside the div, cursor focuses at starting point of the text, not at the place where I clicked.

Please help me out.

**Comment by jasonslyvia** [comment]
Can you provide some snippet or working demo?

**Comment by dineshi2it** [comment]
Please clone this [forked repo](https://github.com/dineshi2it/react-anything-sortable).
Start the app and hit http://0.0.0.0:8722/#/handle

Now, in DemoItem.js, removing `inline: true` , which is in line 26 will work fine.

But I need to pass `inline: true`, as I need to fix the toolbar to the place I need.

**Comment by jasonslyvia** [comment]
After cloning your repo, I fail to figure out how tinymce.js works.

Can you confirm it works without `react-anything-sortable`? By adding three elements in the same page with indentical settings. If so, there might be some event collision( which I don't think probable because I've commented out event handlers in `SortableItemMixin` but the issue remains).

**Comment by dineshi2it** [comment]
So, I removed `mixins: [SortableItemMixin],` and  `this.renderWithSortable` from DemoItem.js. It works as expected.

**Comment by jasonslyvia** [comment]
Yeah I'm sorry kinda busy these days, need to dive into tinymce to figure out why they don't work together.

**Comment by hvergara** [comment]
This is because old version was capturing onSelect event on non-IE browsers.
Fixed in #57
